src/connection.h

```diff
@@ -144,7 +144,8 @@ struct connListener {
     int bindaddr_count;
     int port;
     ConnectionType *ct;
-    void *priv; /* used by connection type specified data */
+    void *priv1; /* used by connection type specified data */
+    void *priv2;
```

Comment:
```suggestion
    void *priv2; /* used by connection type specified data */
```

Reply:
*Two* custom type-specific fields? It looks like a hask. It makes the API a bit messy. Can't the unix socket listener embed both in the same priv field, for example in another allocated struct with two fields (perm and group)?

I don't feel strongly about it though.

Reply:
see https://github.com/valkey-io/valkey/pull/901#pullrequestreview-2237090821

Reply:
OK!